src/main/java/games/strategy/triplea/pbem/AxisAndAlliesForumPoster.java

```diff
@@ -76,7 +76,6 @@ private HttpContext login(final CloseableHttpClient client) throws Exception {
     httpPost.addHeader("Accept", "*/*");
     httpPost.addHeader("Accept-Language", "en-us");
     httpPost.addHeader("Cache-Control", "no-cache");
-    httpPost.addHeader("Content-Type", "application/x-www-form-urlencoded");
```

Comment:
This code is redundant, because a couple lines below we set the entity to `UrlEncodedFormEntity`, which should overwrite this value

Reply:
:+1: Seems to still work as expected.